pkg/ocm/client/testing/mock_model_types.go

```diff
@@ -0,0 +1,30 @@
+package testing
+
+import v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+
+// MockKubeletConfig creates an empty KubeletConfig that can be used in tests. Tests can
+// mutate the KubeletConfig to their requirements via the variadic list of functions
+func MockKubeletConfig(modifyFn ...func(k *v1.KubeletConfig)) (*v1.KubeletConfig, error) {
```

Comment:
Super nitpicky, but I am wondering if we should change naming in this file. 

as generated mock files are prefixed with `mock` would it be worth for other files that are not generated not to follow the same prefix? 

Same for the funcs? Are they mocks or wrappers to builders :thinking: as we are returning the actual object :thinking: 

I am also wondering, would it be worth providing some stock defaults in these funcs, they can be overriden in the modifyFn, it might help reduce some duplicate code across clients when building these objects? 

WDYT?

Reply:
@ciaranRoche Yep, I think that's a good point 👍 I'll find a better name.

I'd be tempted to start out simply returning the empty struct and then over-time move stuff back here if we need to. Was there anything in particular you had in mind though?